bundles/org.openhab.binding.insteon/README.md

```diff
@@ -119,6 +119,7 @@ In order to determine which channels a device supports, you can look at the devi
 | backlightDuration | Number | Back Light Duration |
 | batteryLevel | Number | Battery Level |
 | batteryWatermarkLevel | Number | Battery Watermark Level |
+| beep | Switch | Makes Device Beep |
```

Comment:
Can you change "Makes Device Beep" to "Beep"

Reply:
Done